.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,15 @@
+name: First-Interaction
+
+on: [pull_request, issues]
+
+jobs:
+  first-interaction:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+       
+        pr-message: 'Thank you for submitting this PR. Please double-check to ensure that you have followed our contribution guidelines and update the commit message and title of your Pull Request accordingly. Once you are ready, and all the checks pass, do let us know!'
```

Comment:
is it possible to add the link to contribution guideline here?

Reply:
Let me try

Reply:
Apparently they don't allow links as of now. We might have to proceed ahead with the link functionality for now.

Reply:
Did you try with Markdown? i.e. `[text](url)`